README.md

```diff
@@ -6,11 +6,11 @@ A [bounded context](https://martinfowler.com/bliki/BoundedContext.html) is a sub
 
 The canvas guides you through the process of designing a bounded context by requiring you to consider and make choices about the key elements of its design, from naming to responsibilities, to its public interface and dependencies.
 
-![alt text](resources/bounded-context-canvas-v3.jpeg "The Bounded Context Canvas V3")
+![alt text](resources/bounded-context-canvas-v4.jpeg "The Bounded Context Canvas V4")
 
 ## How to Use
 
-There is no strict rule about the order in which you should fill out the canvas, however, in general it is recommended to start at the top left with the name, move down the left column and then start at the top of the right column.
+Start with the name and description of the canvas to clarify it's reason for existing and key responsibilities in a sentence or two. Then you can fill in the other sections of the canvas in any order. You could design outside-in starting with inbound communication or inside out starting with the business rules and domain language.
```

Comment:
`it's` -> `its`

I think it's fine to say that they have freedom of using whatever order they like, but newcomers often need direction before they can lead on their own. For that reason, I think it might be good to add an example of how the process can look like.

Reply:
Thanks. 

Regarding the order, the default approach is to follow the order things are listed in this readme. I'll update the text to make that clear.

There is also a bigger piece of work to show examples of the process and the end result which people have been asking for. That's on my todo list but I don't have time atm so prefer to defer that to another PR.